content/docs/command-reference/exp/list.md

```diff
@@ -20,9 +20,8 @@ or commit they're based on. This is similar to `dvc exp show --no-pager`, but
 limited to experiment names and with very simple formatting. See also
 `dvc exp run`.
 
-If a working `git_remote` name (e.g. `origin`) or valid Git repo's URL is
-provided, lists experiments in that <abbr>repository</abbr> instead (if any,
-based on the `dvc remote default`).
+If a working `git_remote` name (e.g. `origin`) or Git URL is provided, lists
+experiments in that <abbr>repository</abbr> instead (if any).
```

Comment:
Git URL or file system path to the repo right? Should probably include all the options down here in the description similar to how the `url` arg is explained in https://dvc.org/doc/command-reference/import.

Reply:
But I guess that's out of scope for the PR...